tests/docker/ducktape-deps/addr2line

```diff
@@ -1,6 +1,16 @@
 #!/usr/bin/env bash
 set -e
+
+# Pulls down the addr2line python code for use in DT tests. This doesn't necessarily
+# need to point into our fork, in the case we need upstream improvements we haven't
+# downstreamed yet.
+
+SEASTAR_REPO=scylladb/seastar
+SEASTAR_HASH=0aed36eee620e95c0c8289801824ca52091a0e66
```

Comment:
This looks bogus? Don't think that commit is in that repo yet?

Reply:
right think that commit is in redpanda-data/seastar

Reply:
It "works" due to GH magic:

https://github.com/scylladb/seastar/commit/0aed36eee620e95c0c8289801824ca52091a0e66

but yeah it's wrong, and I meant to put redpanda-data here (but distracted by extracting it out into a var). 

Fixing.